src/core/src/core_logic/PatchInstaller.py

```diff
@@ -88,6 +90,15 @@ def start_installation(self, simulate=False):
 
         self.composite_logger.log("\nInstalled update count: " + str(installed_update_count) + " (including dependencies)")
 
+        endTime = self.env_layer.datetime.datetime_utcnow()
+        patchService = self.package_manager.__class__.__name__
+        machine_details = TelemetryWriter.machine_info
```

Comment:
Always use consistent nomenclature & naming conventions to promote clarity across the code and across code bases, and outcomes of code execution.

So end_time, package_manager_name (only for the variable to disambiguate from the class; but report as package_manager in line 97). machine_info both in name and variable in line 97.

Extrapolate this comment to all code changes - not repeating this everywhere.

Reply:
fixed